sdk/python/jobs/pipelines/1d_pipeline_with_non_python_components/pipeline_with_non_python_components.ipynb

```diff
@@ -143,6 +143,16 @@
     "        file_name=file_name,\n",
     "    ),\n",
     "    outputs={},\n",
+    "    services={\n",
+    "        \"Jupyterlab endpoint\": JobService(job_service_type=\"jupyter_lab\"),\n",
+    "        \"Vscode endpoint\": JobService(job_service_type=\"vs_code\"),\n",
+    "        # \"My_ssh\": JobService(\n",
+    "        # job_service_type = \"ssh\",\n",
+    "        # properties={\n",
+    "        #   \"sshPublicKeys\":\"<add-public-key>\"\n",
```

Comment:
How is the public key replaced/updated on the automated validations?

Reply:
It is a limitation in current preview. But we have a feature where this scenario will be possible. Basically, it will allow user to put a new key in the remote machine on the fly.